docs/docker.md

```diff
@@ -20,25 +38,24 @@ These values will be used by both the database and the app. Now, run either:
 
 #### For Python 2.7
 
-~~~
+```
 $ docker-compose up --build
-~~~
+```
```

Comment:
Changing the styling on this code block is fine; however, this is a duplicate step. We haven't had a build using Python 2.7 in some time.

Reply:
This was another automatic change by the linter. 
Should I just remove this step?

Reply:
Yes.